TFT/src/User/API/Language/language_hu.h

```diff
@@ -278,11 +278,11 @@
     #define STRING_TOUCHSCREEN_ADJUST     "Igazítás"
     #define STRING_MORE                   "Több"
     #define STRING_SCREEN_INFO            "Infó"
-    #define STRING_BG_COLOR               "háttérszín"
-    #define STRING_FONT_COLOR             "betűszín"
+    #define STRING_BG_COLOR               "Háttérszín"
+    #define STRING_FONT_COLOR             "Betűszín"
     #define STRING_PERCENTAGE             "Százalék"
     #define STRING_PERCENTAGE_SPEED       "Sebesség"
-    #define STRING_PERCENTAGE_FLOW        "Hozam"
+    #define STRING_PERCENTAGE_FLOW        "Folyás"
```

Comment:
See above: "Folyás" is a noun made from a verb but the label refers to a quantity.
Wrong correction of a good translation.